opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -314,18 +314,18 @@ message Status {
   // A developer-facing human readable error message.
   string message = 2;
 
-  // For the semantics of status codes see
-  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status
-  enum StatusCode {
-    // The default status.
-    STATUS_CODE_UNSET               = 0;
-    // The Span has been validated by an Application developers or Operator to have
-    // completed successfully.
-    STATUS_CODE_OK                  = 1;
-    // The Span contains an error.
-    STATUS_CODE_ERROR               = 2;
-  };
-
   // The status code.
   StatusCode code = 3;
 }
+
+// For the semantics of status codes see
+// https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status
+enum StatusCode {
```

Comment:
SpanStatusCode? (this is a real question, not sure if it is better)

Reply:
If we do this it will break JSON format (since we also have to rename enum values). I wanted to avoid doing it. Last time we made changes people objected to breaking and we had to do complicated interoperability steps. I'd prefer to avoid doing this again unless we think it is important.